Cargo.toml

```diff
@@ -64,9 +64,9 @@ difference = "1.0"
 tempdir = "0.3"
 
 [features]
-default = []
-unstable = []
-dev = []
+default = ["syntax-highlight"]
+unstable = ["syntax-highlight"]
+dev = ["syntax-highlight"]
```

Comment:
Why specify it in `dev` and `unstable`?  `default` should be sufficient

Reply:
I didn't test with only `default`, I test now :)

Reply:
Thanks